pycode/memilio-simulation/setup.py

```diff
@@ -25,14 +25,14 @@
     # need shared libs so there is one shared log level
     cmake_args=['-DMEMILIO_BUILD_SHARED_LIBS:BOOL=ON'],
     install_requires=[
-        # smaller pandas versions contain a bug that sometimes prevents reading
-        # some excel files (e.g. population or twitter data)
-        'pandas>=2.0.0',
     ],
     extras_require={
         'dev': [
             # smaller numpy versions cause a security issue, 1.25 breaks testing with pyfakefs
             'numpy>=1.22,<1.25',
+            # smaller pandas versions contain a bug that sometimes prevents reading
+            # some excel files (e.g. population or twitter data)
+            'pandas>=2.0.0',
```

Comment:
is pandas actually required by the simulation package? looks like this was just never amended when the epidata and simulation packages where split.

Reply:
Pandas is used in the test and examples of simulation. Beccause of that pandas is part of the .[dev] target and not the base install requirements.

Reply:
I stumbled over that before, maybe it's not ideal to have that large dependency just for tests, but certainly not to be solved here.